cpu/stm32/include/periph/f0/periph_cpu.h

```diff
@@ -23,6 +23,21 @@
 extern "C" {
 #endif
 
+/**
+ * @brief   Starting address of the ROM bootloader
+ *          see application note AN2606
+ */
+#if defined(CPU_LINE_STM32F030x4) || defined(CPU_LINE_STM32F030x6) || \
```

Comment:
```suggestion
#ifdef DOXYGEN
#define STM32_LOADER_ADDR
#elif defined(CPU_LINE_STM32F030x4) || defined(CPU_LINE_STM32F030x6) || \
```

Adding an extra empty definition just for Doxygen is often used to ensure that the macro turns up in the generated documentation while avoiding to lure users into believing that whichever value in the definition makes it into the documentation is the "correct" value for all STM32 MCUs.

Update: How about adding the the doxygen entry to `cpu/stm32/include/periph_cpu.h` instead, so that it only needs to be documented once?

```C
#ifdef Doxygen
/**
 * @brief   Starting address of the ROM bootloader
 *          see application note AN2606
 */
#define STM32_BOOTLOADER_ADDR 
#endif
```

Reply:
This also matches the other defines.